rules/windows/defense_evasion_suspicious_managedcode_host_process.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious Managed Code Hosting process which could indicate code injection or other form of suspicious code
+execution. Verify process details such as command line, parent process, and network connections.
```

```suggestion
Identifies a suspicious managed code hosting process which could indicate code injection or other form of suspicious code
execution. 
```
Would move to `note` - 
> Verify process details such as command line, parent process, and network connections.